platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/renderer/MapRenderer.java

```diff
@@ -30,9 +30,9 @@
 
   // Holds the pointer to the native peer after initialisation
   private long nativePtr = 0;
-
   private double expectedRenderTime = 0;
   private MapboxMap.OnFpsChangedListener onFpsChangedListener;
+  protected boolean hasSurface;
```

Comment:
This can be `private` now, or `GLSurfaceViewMapRenderer#requestRenderer` should use a field reference instead of a getter.